internal/cli/atlas/backup/restores_list.go

```diff
@@ -38,14 +38,18 @@ func (opts *RestoresListOpts) initStore() error {
 	return err
 }
 
+var restoreListTemplate = `ID	SNAPSHOT	CLUSTER	TYPE	EXPIRE{{range .Results}}
```

Comment:
[nit-grammar] missing an `S` but I'd keep the `AT` to indicate is a date
```suggestion
var restoreListTemplate = `ID	SNAPSHOT	CLUSTER	TYPE	EXPIRES AT{{range .Results}}
```